fontbe/src/orchestration.rs

```diff
@@ -0,0 +1,150 @@
+//! Helps coordinate the graph execution for BE
+
+use std::{collections::HashSet, fs, path::Path, sync::Arc};
+
+use fontdrasil::orchestration::{Acl, Cache, Work, MISSING_DATA};
+use write_fonts::FontBuilder;
+
+use crate::{error::Error, paths::Paths};
+
+// Unique identifier of work. If there are no fields work is unique.
+// Meant to be small and cheap to copy around.
+#[derive(Clone, Debug, PartialEq, Eq, Hash, PartialOrd, Ord)]
+pub enum WorkIdentifier {
+    Features,
+    Glyph(String),
```

Comment:
If this is intended to be cheap to copy around, I don't think it should be a `String`, which is expensive to clone.

not necessary for this PR, but I am confident that you will want to use some other type for this eventually, and it's probably worth doing this sooner rather than later. Even if all you do is create a newtype like `GlyphName(String)` for now, then it will be much easier for you to refactor this down the road.

Reply:
It's on my todo list to make it a SmolStr. I think that's what fea-rs uses as well?

Filed https://github.com/googlefonts/fontmake-rs/issues/86 to increase the odds of remembering.